agent/src/adapter/brief.py

```
@@ -0,0 +1,309 @@
+                "content": "Conversation so far:\n"
+                + "\n".join(f"{m['role']}: {m['content']}" for m in transcript),
+            },
+        ]
+
+        error: str | None = None
+        for attempt in ("first", "repair"):
+            if attempt == "repair":
+                messages = messages + [
+                    {"role": "assistant", "content": error or ""},
+                    {"role": "user", "content": _REPAIR.format(error=error)},
+                ]
```

Ruff (0.16.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)